code/game/machinery/pipe/pipe_construction.dm

```diff
@@ -537,18 +537,16 @@
 	item_state = "buildpipe"
 	w_class = WEIGHT_CLASS_BULKY
 
-/obj/item/pipe_meter/attackby(obj/item/W, mob/user, params)
-	if(!iswrench(W))
-		return ..()
-
+/obj/item/pipe_meter/wrench_act(mob/living/user, obj/item/I)
 	if(!locate(/obj/machinery/atmospherics/pipe, loc))
-		to_chat(user, "<span class='warning'>You need to fasten it to a pipe</span>")
+		to_chat(user, "<span class='warning'>You need to fasten it to a pipe.</span>")
 		return TRUE
 
 	new /obj/machinery/atmospherics/meter(loc)
-	playsound(loc, W.usesound, 50, 1)
+	W.play_tool_sound(src)
```

Comment:
```suggestion
	I.play_tool_sound(src)
```

Reply:
I swear I tested this, but in my defense, I was on an airplane